src/common/CommandLine.cs

```diff
@@ -1746,18 +1746,18 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash;
+        private bool _qualifiersUseOnlyDash = true;
```

Comment:
FYI, You can pass ?w=1 to the diff view to have it ignore white-space only changes.